rubygem-foreman_gutterball.spec

```diff
@@ -57,28 +57,35 @@ Documentation for %{name}
 
 %prep
 %setup -q -c -T
-mkdir -p .%{gemdir}
+mkdir -p .%{gem_dir}
 %{?scl:scl enable %{scl} "}
-gem install --local --install-dir .%{gemdir} \
-            --force %{SOURCE0}
+gem install --local --install-dir .%{gem_dir} \
+            --force %{SOURCE0} --no-rdoc --no-ri
 %{?scl:"}
 
 %build
 
 %install
-mkdir -p %{buildroot}%{gemdir}
-cp -pa .%{gemdir}/* \
-        %{buildroot}%{gemdir}/
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{foreman_bundlerd_dir}
+
+cat << EORUBY > %{buildroot}%{foreman_bundlerd_dir}/foreman_gutterball.rb
+gem 'foreman_gutterball'
+EORUBY
```

Comment:
The foreman_bunderld_file macro below this should handle this part for you (and the mkdir -p) - https://github.com/theforeman/foreman-packaging/blob/rpm/develop/foreman/foreman.spec#L525